java/core/src/test/java/sleeper/core/table/TableIdGeneratorTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022-2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sleeper.core.table;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TableIdGeneratorTest {
+
+    @Test
+    void shouldGenerate32BitHexId() {
+        assertThat(TableIdGenerator.fromRandomSeed(0).generateString())
+                .isEqualTo("60b420bb");
```

Comment:
Is it worth testing that a second call to the same `TableIdGenerator` gives a different string to the first one? There's nothing to stop someone reusing the same `TableIdGenerator` many times.

Reply:
Done